I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/nerkk_cap4.rpy", line 372, in script
    professor "Onde ele está?!!!"
Exception: Sayer 'professor' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/nerkk_cap4.rpy", line 372, in script
    professor "Onde ele está?!!!"
  File "C:\Users\thoma\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 2586, in execute
    Say.execute(self)
  File "C:\Users\thoma\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 583, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\thoma\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 472, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'professor' is not defined.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
Kaua 1.0
Sat May 17 19:14:36 2025
